gobblin-core/src/main/java/gobblin/source/extractor/utils/InputStreamCSVReader.java

```diff
@@ -49,7 +49,7 @@ public InputStreamCSVReader(BufferedReader input) {
 
   public InputStreamCSVReader(String input) {
     this(new InputStreamReader(new ByteArrayInputStream(input.getBytes()),
-        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)));
+        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)), ',', '\"');
```

Comment:
LGTM. Original code is not handling if the text enclosed in double quotes. This change is to handle this case also.